discord_bots/cogs/category.py

```diff
@@ -0,0 +1,337 @@
+import logging
+
+from discord import Colour, Embed, Interaction, app_commands
+from discord.ext.commands import Bot
+from sqlalchemy.orm.exc import NoResultFound
+from sqlalchemy.orm.session import Session as SQLAlchemySession
+
+from discord_bots.checks import is_admin_app_command, is_command_channel
+from discord_bots.cogs.base import BaseCog
+from discord_bots.models import Category, PlayerCategoryTrueskill, Queue, Session
+from discord_bots.utils import default_sigma_decay_amount, buildCategoryOutput
+
+_log = logging.getLogger(__name__)
+
+
+class CategoryCommands(BaseCog):
+    def __init__(self, bot: Bot):
+        super().__init__(bot)
+
+    group = app_commands.Group(name="category", description="Category commands")
+    
+    @group.command(name="clearqueue", description="Remove category from queue")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(queue_name="Name of existing queue")
+    async def clearqueuecategory(self, interaction: Interaction, queue_name: str):
+        session: SQLAlchemySession
+        with Session() as session:
+            try:
+                queue: Queue = (
+                    session.query(Queue).filter(Queue.name.ilike(queue_name)).one()
+                )
+            except NoResultFound:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description=f"Could not find queue **{queue_name}**",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+            else:
+                queue.category_id = None
+                session.commit()
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description=f"Queue **{queue.name}** category cleared",
+                        colour=Colour.green(),
+                    ),
+                    ephemeral=True,
+                )
+
+    @group.command(name="create", description="Create a new category")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(name="Name of new category")
+    async def createcategory(self, interaction: Interaction, name: str):
```

Comment:
If you're looking at my dev bot, that'll be because it's synced to my `RotationConfigure` fork at the moment, which also has `CategoryConfigure` & `MapConfigure` within it.

As a part of `CategoryConfigure` this command was removed in favour of `/category configure <category_name>`.
That is not the case for this PR though, and I have tested this successful.